### PR TITLE
Adds more plushies

### PR DESCRIPTION
**Pull request by Amelia0010**
### Intent of your Pull Request

Ports the plushies from these two PRs: https://github.com/tgstation/tgstation/pull/30751 & https://github.com/tgstation/tgstation/pull/30710. 

Also adds a plushie crate and removes the carp plushie from the toy crate.

These are the plushies:
image

Also includes an inorix plushie:
image

#### Changelog

:cl: TehZombehz, ninjanomnom & KMC2000
rscadd: The station has been supplied with more plushies! You can either find them in arcade machines or buy the new plushie crate!
/:cl:

### CODE REVIEW

**Comment by Partheo** [comment]
I think they should be outlined in a darker color, since they look like stickers currently.

Otherwise, cool and good PR. Cute.

**Comment by Ratvar** [comment]
Make Nar'Sie and Ratvar convert all plushies into their, uh, their godly alternative?

**Comment by Jc1469** [comment]
wurs inorix plush

**Comment by Amelia0010** [comment]
the fuck is an inorix

**Comment by Nickvr628** [comment]
Not a huge fan of the art style, especially those on the top row.

**Comment by Nickvr628** [comment]
Not a huge fan of the art style, especially those on the top row.

**Comment by ShadowDeath6** [comment]
Please credit the original content creators in the changelog.

Nar'sie and bubblegum look fantastic, I like the lizard but I don't really like the way the nuke op is colored. 

also I have no idea what the thing on the top right is. It kind of looks like a slime but its eyes are(look?) reflective which is kind of strange. I also think it would look nicer if it was gray, or at least muted, since it only looks like a slime in body shape and mouth.

**Comment by ShadowDeath6** [comment]
also these should really be in the gift shop for obvious reasons

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -1066,31 +1066,108 @@
...
+	icon_state = "plushvar"
+
+//plushvar conversion
+/obj/item/toy/plushie/plushvar/ratvar_act()
+	return
+
+//narsie plush
+/obj/item/toy/plushie/narplush
+	name = "nar-sie plushie"
+	desc = "A small stuffed doll of the elder god nar'sie. Who thought this was a good children's toy?"
```
Comment (ShadowDeath6) [inline code/game/objects/items/toys.dm]: names are proper nouns, capitalize this and ratvar's

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -1443,13 +1443,25 @@
...
-	                /obj/item/toy/carpplushie,
 	                /obj/item/weapon/coin/antagtoken,
 	                /obj/item/stack/tile/fakespace/loaded,
 	                /obj/item/weapon/gun/projectile/shotgun/toy/crossbow,
 	                /obj/item/toy/redbutton)
 	crate_name = "toy crate"
 
+/datum/supply_pack/misc/plushies
+	name = "Plushie Crate"
+	cost = 2000
```
Comment (ShadowDeath6) [inline code/modules/cargo/packs.dm:1454]: this seems a bit on the expensive side although I'm not certain

Comment (Amelia0010) [reply code/modules/cargo/packs.dm:1454]: The toy crate is at 5000, which is what I lowered it from. If you have a price in mind i'll just change it to that

**Comment by ShadowDeath6** [comment]
can you put some of them in the gift shop

**Comment by Amelia0010** [comment]
They're all in the Clerk's gift vendor, under random plushie. Do you mean the actual shop room itself?

**Comment by ShadowDeath6** [comment]
I meant to give them to the clerk in some way, what you did is fine

**Pull request merged**